src/cdc_uart.c

```diff
@@ -66,80 +74,102 @@ void cdc_uart_init(void) {
     gpio_set_dir(PICOPROBE_UART_DTR, GPIO_OUT);
     gpio_put(PICOPROBE_UART_DTR, 1);
 #endif
+#if (CDC_UARTS > 1)
```

Comment:
Should there be a compile-time error or warning if `CDC_UARTS` is set to a value larger than 2?

Reply:
My thinking was that user provides or omits EXTRA_UARTS boolean, then CDC_UARTS only really settable by configure: 1 by default, 2 if EXTRA_UARTS present. again thoughts were that separating the boolean from the count allowed future expansion by not excluding additional software uarts bridged to CDC.
Will add that warning once you give me the go/no-go on other two points...